### ISSUE TITLE
Add info on landing page about how you should record/convert ROS 2 files on the same distro to avoid issues

### ISSUE THREAD

**Issue by esthersweon** [foxglove/mcap#403]
**Description**
- Add info on landing page about how you should record/convert ROS 2 files on the same distro to avoid issues

**Comment by esthersweon** [comment]
@amacneil @defunctzombie - this seems more like a callout about how foxglove CLI works than about how MCAP the file format does. 

I propose we call this out in the [CLI repo](https://github.com/foxglove/mcap/tree/main/go/cli/mcap), instead of the MCAP website like was originally intended when opening this ticket.

**Comment by amacneil** [comment]
I agree it's a caveat of the CLI rather than MCAP itself. So we should note it in the CLI docs.

However, I think we should move the CLI docs out of that readme into our new docs site.
